<?xml version="1.0"?>
<launch>
    <node ns="iiwa" 
          pkg="pkg_ijcai2019" 
          type="sim_scene" 
          name="sim_scene" 
          output="screen" respawn="false"/>

    <rosparam param="changkyu/pkg_ijcai2019/sim_scene" command="load" file="$(find pkg_ijcai2019)/config/input_scene.yaml" />
</launch>
